modules/grpc/otel/otel-protobuf-formatter.cpp

```diff
@@ -515,6 +515,48 @@ ProtobufFormatter::set_syslog_ng_nv_pairs(LogMessage *msg, LogRecord &log_record
                       &user_data);
 }
 
+void
+ProtobufFormatter::set_syslog_ng_address_attrs(GSockAddr *sa, KeyValueList *address_attrs, bool include_port)
+{
+  gchar addr_buf[32];
+  gsize addr_len;
+
+  if (!g_sockaddr_get_address(sa, (guint8 *) addr_buf, sizeof(addr_buf), &addr_len))
+    return;
+
+  KeyValue *addr = address_attrs->add_values();
+  addr->set_key("addr");
+  addr->mutable_value()->set_bytes_value(addr_buf, addr_len);
```

Comment:
We need proper serialization here, sending raw `sockaddr` structures are not portable as far as I know.

This patch also contains an ivykis submodule update (probably by accident).

Reply:
Right, I confused sockaddr_in with in_addr.

LGTM then (we need to remove the ivykis bump from the commit).